include/deal.II/lac/petsc_vector_base.h

```diff
@@ -1271,6 +1308,66 @@ namespace PETScWrappers
       }
   }
 
+  template <class Archive>
+  inline void
+  VectorBase::save(Archive &ar, const unsigned int) const
+  {
+    // forward to serialization function in the base class.
+    ar &static_cast<const Subscriptor &>(*this);
+    ar &size();
+    ar &local_range();
+
+    const PetscScalar *array = nullptr;
+    int                ierr  = VecGetArrayRead(*this, &array);
+    AssertThrow(ierr == 0, ExcPETScError(ierr));
+
+    boost::serialization::array_wrapper<const double> velocity_wrapper(
```

Comment:
Maybe a more generic name is more appropriate 😉

Reply:
Yes, this is worth fixing.